client/foundries.go

```diff
@@ -249,6 +249,20 @@ func NewApiClient(serverUrl string, config Config, caCertPath string) *Api {
 	return &api
 }
 
+func getResponse(resp *[]byte, err error, runName string) (string, error) {
+	if err != nil {
+		return "", err
+	}
+	type PatchResp struct {
+		JobServUrl string `json:"web-url"`
```

Comment:
I think this needs to be:
~~~
		JobServUrl string `json:"url"`
		WebUrl string `json:"web-url"`
~~~

Then we return this tuple to the caller.

Reply:
Do you mean jobserve-url? I thought about it too. But a caller, in this case, is a `fioctl targets <subcommand>` command, the question is what a user can do with the URL that points to  API. Maybe it's useful for the case when fioctl is used programmatically by some other tool.

Reply:
isn't the caller is this function printing the value out and then passing it tail function? ie - one should be a web-url and the other should be an API url

Reply:
yes, I just thought that the web-url like this https://ci.foundries.io/projects/msul-dev01/lmp/builds/439/assemble-system-image/artifacts/console.log is also good enough for the tailing.
Anyway, I think, it makes sense to return both URLs - it is just more flexible and fulfills more use cases.